deploy/stacks/pipeline.py

```diff
@@ -765,7 +775,11 @@ def set_cloudfront_stage(self, target_env):
                 role=self.expanded_codebuild_role.without_policy_updates(),
                 vpc=self.vpc,
             ),
-            self.cognito_config_action(target_env),
+        )
+        if target_env.get('custom_auth', None) is None:
+            front_stage_actions = (
+                *front_stage_actions,
+                self.cognito_config_action(target_env),
         )
         if target_env.get('enable_cw_rum', False):
```

Comment:
should we also a check here (L 784) similar to what we pass in BackendStage `target_env.get('enable_cw_canaries', False) and target_env.get("custom_auth", None) == None,` so cw rum actions only occur if cw rum enabled AND custom auth not specified

Reply:
Yes, missed this part, Was of the opinion that Cognito stack is not needed for this one. But turns out thet cw_rum needs cognito identity pool to be enabled. 

Making `enable_cw_rum ` config dependent  on `custom_auth ` config and adding this check whereever needed